tests/test_lowlevel.py

```diff
@@ -209,7 +209,6 @@ def test_use_proxy_from_environment(httpbin, var, scheme):
 def test_redirect_rfc1808_to_non_ascii_location():
     path = u'š'
     expected_path = quote(path.encode('utf8')).encode('ascii')
```

Comment:
It seems like it might be more resilient/explicit to hard code the expected path here? It would divest us from relying on `quote` in the test, saves us an import, and hopefully clear up this line a bit.

At the very least, a comment on what we're doing here seems good.